synfig-studio/src/gui/trees/layertreestore.h

```diff
@@ -155,7 +154,7 @@ class LayerTreeStore : virtual public Gtk::TreeStore
 	void set_gvalue_tpl(Glib::ValueBase& value, const T &v, bool use_assign_operator = false) const;
 
 	virtual void  set_value_impl (const Gtk::TreeModel::iterator& row, int column, const Glib::ValueBase& value);
-	virtual void  get_value_vfunc (const Gtk::TreeModel::iterator& iter, int column, Glib::ValueBase& value)const;
+	virtual void  get_value_vfunc (const Gtk::TreeModel::iterator& iter, int column, Glib::ValueBase& value) const override;
```

Comment:
You taught me `override` does not require the `virtual` keyword :)
Should we keep it?

Reply:
Fixed.
I don't remember why I didn't fix it. Maybe just missed it.